Gordon360/AuthorizationFilters/StateYourBusiness.cs

```diff
@@ -564,7 +564,7 @@ private bool canUpdate(string resource)
                             if (applicationID.HasValue && applicationID == requestedApplicationID)
                             {
                                 var editorUsername = housingService.GetEditorUsername((int)applicationID);
-                                if (editorUsername == user_name)
+                                if (editorUsername.ToLower() == user_name.ToLower())
```

Comment:
The use of `ToLower()` is actually extremely common for things like this, and we should be using it more often. 
Good catch